CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - [...]
 - **[UPDATE]** `Storybook` version `6.1.15`
+- **[NEW]** Add `Showcase`
```

Comment:
I would call it `Marquee`

Reply:
Now that we know it actually slides, I'll go for Marquee (will change when it officially makes it into the Design System)